### PR TITLE
Stuns do not affect tail wagging, but paralyzes still do

### PR DESCRIPTION
**Pull request by Sniblet**
I think someone accidentally did the exact opposite of what an inoffensive QoL PR proposed to do yesterday. I can’t contact them right now so I’m just offering to correct it for them

Most people don’t try to wag all round and so don’t know how annoying this is. Let me solve my own problem.
If you want to grief felinids, this only affects: sigils of submission, neural shock, vampire glare, seeing the singularity without mesons, CNS EMP, the Voice of God yelling STOP, and climbing tables. As you will not be using any of these to force waggers to type *wag again, this does not affect you.

# Document the changes in your pull request

If a tailed creature is paralyzed (flash, electrocutions, slips, beepsky), they will not be able to start wagging and will stop wagging if they are.
If they are stunned (sigil of submission, CNS EMP, climbing tables), their tail is not paralyzed, so they can still start wagging and will continue to wag if they are.

# Changelog

:cl:  
rscadd: added justice to the world
bugfix: Things that don’t paralyze you don’t paralyze your tail
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Deceptive PR. Just write that it is a revert of #13343 instead of acting like this is something new

**Comment by Sniblet** [comment]
It’s not a revert, it’s a compromise. I would revert if I thought that would go through, but since it wouldn’t, I kept in the stun effect and am redoing #13341 with the distinction between stuns and paralyzes in mind.
If the description doesn’t make this clear, you can look at the files changed. I edited your work, I didn’t undo it. Not a revert.

**Comment by Marmio64** [comment]
Get a proper pr name

**Comment by Hopekz** [comment]
> It’s not a revert, it’s a compromise.

It's kind of a never ending cycle with the instant "react to a PR" PR's.
 Should we expect a compromise PR right after this one is merged to compromise on these changes too?

**Comment by Sniblet** [comment]
Well, I'm sorry, Hopekz, but the PR was my idea and Baiomu closed it without saying anything to me. I felt safe in reproposing it at least until someone was able to make a coherent point regarding why making it easier for a character to keep their tail wagging throughout the round was worse for the game than making it even more inconvenient.

Until a decent argument comes around, I can only assume that the only reason people have been opposing this is because it can make playing felinid more pleasant. It's funny when you're oppressing the degenerate cats ingame, it's just toxic when you do it on the git.

Since I don't expect anyone to care about that after the first PR was silently closed with 11 thumbs-down, I gave a little by leaving in the ability for paralyzes to forcibly stop wagging.

If anyone decides to try and go back on what this PR does, I will be shocked at the display of bad faith. To be clear, _stuns_ as opposed to paralyzes are where your character's movement and ability to use their hands are suspended, but they don't fall down. You can count on your fingers the number of things in the entire game that do that: sigils of submission, neural shock, vampire glare, seeing the singularity without mesons, CNS EMP, the Voice of God yelling STOP, and climbing tables are all I can call to mind. This PR, in its finished form, would only take away those things' abilities to grief waggers.
You still have the unlimited ability flash them so they have to stop for five seconds (up from one before your fix), at the cost of half of what I wanted, which was to not have wagging interrupted when I slip. Rejoice.

**Comment by Hopekz** [comment]
> Well, I'm sorry, Hopekz

I forgive you

**Comment by Sniblet** [comment]
Okay I think this does the intended thing from the title, will test because this isn't copypasta so I don't know
I might have a way to also do this for slips, but I don't know if it will add an unintended amount of jank and I should probably do it separately because this is already controversial (please see previous rant for why it shouldn't be)

**Comment by Sniblet** [comment]
Works as described

**Comment by Sniblet** [comment]
It does not do the intended
If anybody knows how to make paralyzes stop wagging without stuns also doing it, speak up anytime

**Comment by Sniblet** [comment]
"could add an arg that differentiates if its a stun or whatever and check that, or you could make a separate proc for each of them and copy the code for spec_stun"
someone translate

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1969,6 +1969,7 @@ GLOBAL_LIST_EMPTY(mentor_races)
 		ToggleFlight(H)
 		flyslip(H)
 	. = stunmod * H.physiology.stun_mod * amount
+/datum/species/proc/spec_paralyze(mob/living/carbon/human/H,amount)
```
Comment (nmajask) [inline code/modules/mob/living/carbon/human/species.dm]:
```suggestion
```
Remove this if you're not going to use it

**Comment by Sniblet** [comment]
Tested, works as described

**Comment by ToasterBiome** [comment]
> Well, I'm sorry, Hopekz, but the PR was my idea and Baiomu closed it without saying anything to me.

For the record, it was my PR and I closed it because I merged another PR that invalidates it. Feel free to PM me with any queries you might have instead of name dropping me

File: code/modules/mob/living/carbon/human/status_procs.dm [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 /mob/living/carbon/human/Paralyze(amount, updating = TRUE, ignore_canstun = FALSE)
 	amount = dna.species.spec_stun(src, amount)
```
Comment (nmajask) [inline code/modules/mob/living/carbon/human/status_procs.dm:11]:
```suggestion
	if(!dna || !dna.species)
		return ..()
	amount = dna.species.spec_stun(src, amount)
```
Add a check to make sure dna and dna.species aren't null

**Review by nmajask — CHANGES_REQUESTED** [review]

**Comment by JamieD1** [comment]
A stun is just as bad as a paralyze, I don't get this point of this PR. Tail wagging should be stopped with any sort of immobilize.

**Comment by Sniblet** [comment]
> A stun is just as bad as a paralyze, I don't get this point of this PR. Tail wagging should be stopped with any sort of immobilize.

I wrote paragraphs about why this was wrong
Why is it so hard to make my life a little easier without harming anyone else

**Pull request closed without merging**